.github/workflows/ci.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
windows may have some issues with prettier formatting and linting
If so we need to copy over this snipped: 

https://github.com/prettier/plugin-pug/blob/6b88a4fb2c49bb6dd321c9887b36574383097366/.github/workflows/ci.yml#L19-L23

Reply:
Nevermind, cause I only run lint on linux+node14 it's not a problem here